modern/src/utils.ts

```diff
@@ -442,3 +443,42 @@ if (typeof document !== "undefined") {
 export function getMousePosition() {
   return mousePosition;
 }
+
+export function imageAttributesFromNode(node: XmlNode): Array<string> {
```

Comment:
I moved this here because its so similar to the new function  (but I dont think I can combine them), and they will needed to be added to at the same time as we add more components so wanted them to be next to each other